src/profiler.ts

```diff
@@ -111,7 +111,8 @@ function getServerResponseBackoff(body: object): number | undefined {
         item.retryDelay &&
         typeof item.retryDelay === 'string'
       ) {
-        const backoffMillis = parseDuration(item.retryDelay)!;
+        const retryDelay: `${number}s` = item.retryDelay;
```

Comment:
This should be safe because [JSON encoding of protobuf duration](https://protobuf.dev/programming-guides/json/#:~:text=Generated%20output%20always%20contains%200%2C%203%2C%206%2C%20or%209%20fractional%20digits%2C%20depending%20on%20required%20precision%2C%20followed%20by%20the%20suffix%20%22s%22.%20Accepted%20are%20any%20fractional%20digits%20(also%20none)%20as%20long%20as%20they%20fit%20into%20nano%2Dseconds%20precision%20and%20the%20suffix%20%22s%22%20is%20required.) is always suffixed with "s".

Reply:
Might be worth adding that info as a comment for future readers.

Reply:
Added